hack/local-up-operator.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Chaos Mesh Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROOT=$(unset CDPATH && cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+cd $ROOT
+
+source "${ROOT}/hack/lib.sh"
+
+function usage() {
+    cat <<'EOF'
+This commands run chaos-mesh in Kubernetes.
+
+Usage: hack/local-up-operator.sh [-hd]
+
+    -h      show this message and exit
+    -i      install dependencies only
+
+Environments:
+
+    PROVIDER              Kubernetes provider. Defaults: kind.
+    CLUSTER               the name of e2e cluster. Defaults to kind for kind provider.
+    KUBECONFIG            path to the kubeconfig file, defaults: ~/.kube/config
+    KUBECONTEXT           context in kubeconfig file, defaults to current context
+    NAMESPACE             Kubernetes namespace in which we run our chaos-mesh.
+    DOCKER_REGISTRY       image docker registry
+    IMAGE_TAG             image tag
+    SKIP_IMAGE_BUILD      skip build and push images
+
+EOF
+}
+
+
+installOnly=false
+while getopts "h?i" opt; do
+    case "$opt" in
+    h|\?)
+        usage
+        exit 0
+        ;;
+    i)
+      installOnly=true
+        ;;
+    esac
+done
+
+PROVIDER=${PROVIDER:-kind}
+CLUSTER=${CLUSTER:-}
+KUBECONFIG=${KUBECONFIG:-~/.kube/config}
+KUBECONTEXT=${KUBECONTEXT:-}
+DOCKER_REGISTRY_PREFIX=${DOCKER_REGISTRY_PREFIX:-localhost:5000}
```

Comment:
Is it necessary to set the default value to "localhost:5000" here?

Reply:
I think local registry always is localhost:5000, so we can set it as default value.